changelog.dd

```diff
@@ -327,6 +329,15 @@ static assert( isFinal!(C.ff));
 -------
 )
 
+$(LI $(LNAME2 generate, `std.range.generate` fixed to be a proper range.)
```

Comment:
Most expect ranges to behave this way, the updating of the rules is just a formal presentation of what everyone already knows. So I don't think a link is necessary.